src/nile/common.py

```diff
@@ -28,25 +28,35 @@
     # subject to change
     "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf"
 )
+DEFAULT_GATEWAYS = {
+    "localhost": "http://127.0.0.1:5050/",
+    "goerli2": "https://alpha4-2.starknet.io",
+    "integration": "https://external.integration.starknet.io",
+}
 
 
 def get_gateways():
     """Get the StarkNet node details."""
-    try:
+    if os.path.exists(NODE_FILENAME):
         with open(NODE_FILENAME, "r") as f:
             gateway = json.load(f)
```

Comment:
```suggestion
            custom_gateways = json.load(f)
```

The name is a bit confusing being possible to have multiple custom gateways in the `node.json` file.